packages/kiota_serialization_form/lib/src/form_serialization_writer.dart

```diff
@@ -81,7 +81,7 @@ class FormSerializationWriter implements SerializationWriter {
         valueNames.write(',');
       }
 
-      valueNames.write(value.name.toFirstCharacterLowerCase());
+      valueNames.write(value.name.toLowerCase());
```

Comment:
my suggestion was to avoid doing anything with casing at runtime

Reply:
Gets handled by `EnumRegistry`. Internally it compares using the lowercase variant. I'm afraid there is no way around it